doc/modules/cross_decomposition.rst

```diff
@@ -23,19 +20,170 @@ Cross decomposition algorithms find the fundamental relations between two
 matrices (X and Y). They are latent variable approaches to modeling the
 covariance structures in these two spaces. They will try to find the
 multidimensional direction in the X space that explains the maximum
-multidimensional variance direction in the Y space. PLS-regression is
-particularly suited when the matrix of predictors has more variables than
-observations, and when there is multicollinearity among X values. By contrast,
-standard regression will fail in these cases.
+multidimensional variance direction in the Y space. In other words, PLS
+projects both `X` and `Y` into a lower-dimensional subspace such that the
+covariance between `tranformed(X)` and `transformed(Y)` is maximal.
+
+PLS draws similarities with Principal Component Regression, where the samples
```

Comment:
```suggestion
PLS draws similarities with Principal Component Regression (PCR), where the samples
```

Reply:
do we say somewhere what PCR does (pipeline PCA -> Regressor) or do we provide a ref?